packages/augur-ui/src/modules/portfolio/components/portfolio-view/my-bets.tsx

```diff
@@ -73,35 +110,69 @@ export const MyBets = () => {
             <SquareDropdown
               options={MY_BETS_VIEW_BY}
               defaultValue={MY_BETS_VIEW_BY[0].value}
-              onChange={selected => setState({ ...state, viewBy: selected })}
-              minimalStyle
-            />
-          </span>
-          <span>
-            Market Status:{' '}
-            <SquareDropdown
-              options={MY_BETS_MARKET_STATUS}
-              defaultValue={MY_BETS_MARKET_STATUS[0].value}
               onChange={selected =>
-                setState({ ...state, marketStatus: selected })
+                setState({
+                  ...state,
+                  viewBy: selected,
+                  rows:
+                    MY_BETS_VIEW_BY[selected].label === EVENT
+                      ? MOCK_GAMES_DATA
+                      : MOCK_OUTCOMES_DATA,
+                })
               }
               minimalStyle
             />
           </span>
+          {showEvents ? (
+            <span>
+              Market Status:{' '}
+              <SquareDropdown
+                options={MY_BETS_MARKET_STATUS}
+                defaultValue={MY_BETS_MARKET_STATUS[0].value}
+                onChange={selected =>
+                  setState({ ...state, marketStatus: selected })
+                }
+                minimalStyle
+              />
+            </span>
+          ) : (
+            <span>
+              Bet Date:{' '}
```

Comment:
can we use a template string to avoid having to add `{' '}`? I forget if that works I just hate those inserted spaces :)